test/acquisition/test_proximal.py

```diff
@@ -179,6 +179,24 @@ def test_proximal(self):
                     ExpectedImprovement(bad_single_task, 0.0), proximal_weights
                 )
 
+            # test a multi-output SingleTaskGP model
+            train_X = torch.rand(5, 3, device=self.device, dtype=dtype)
+            train_Y = torch.rand(5, 2, device=self.device, dtype=dtype)
+
+            multi_output_model = (
+                SingleTaskGP(train_X, train_Y).to(device=self.device).eval()
```

Comment:
I don't think the `eval` should be necessary here - when called in `acq`, the `posterior` call of the model should automatically make sure that the model is in eval mode.